src/components/DpDataTable/DpResizableColumn.vue

```diff
@@ -2,7 +2,7 @@
   <th
     v-tooltip="headerField.tooltip || headerField.label"
     ref="resizableColumn"
-    class="c-data-table__resizable"
+    class="c-data-table__resizable break-words"
```

Comment:
nitpick: we may also use tailwind here I guess, and that would be like that:
`hyphens-auto`

https://tailwindcss.com/docs/hyphens#auto

Reply:
I'm not sure what you mean here. `break-words` is a tailwind class. It is extended in the override in tailwind.config.js with `hyphens: auto`: 
`        '.break-words': {
          'word-wrap': 'break-word',
          'word-break': 'break-word',
          'hyphens': 'auto'
        },`